src/node.rs

```diff
@@ -278,9 +265,31 @@ impl Taffy {
         Ok(self.nodes[node].is_dirty)
     }
 
-    /// Updates the stored layout of the provided `node` and its children
+    /// Updates the stored layout of the provided `node` and its children with measure
+    pub fn compute_measured_layout(
+        &mut self,
+        node: Node,
+        measure: &impl Measure,
+        size: Size<Option<f32>>,
+    ) -> Result<(), TaffyError> {
+        self.compute(node, size, measure);
+        Ok(())
+    }
+}
+
+struct NoopMeasure();
+
+impl Measure for NoopMeasure {
+    fn measure(&self, _node: Node, _size: Size<Option<f32>>) -> Size<f32> {
+        Size::ZERO
+    }
+}
+
+impl Taffy {
+    /// Updates the stored layout of the provided `node` and its children with no measurements
     pub fn compute_layout(&mut self, node: Node, size: Size<Option<f32>>) -> Result<(), TaffyError> {
```

Comment:
This method might be misleading because it can be called regardless whether or not Taffy has any nodes that require measurements. In other words, the type doesn't capture this requirements any more, thus maybe require `Measure` trait implementation for all cases?